src/generator/AutoRest.Ruby.Azure/AzureRubyCodeGenerator.cs

```diff
@@ -93,14 +93,25 @@ private void AddRubyPageableMethod(ServiceClient serviceClient)
                     PageableExtension pageableExtension = JsonConvert.DeserializeObject<PageableExtension>(method.Extensions[AzureExtensions.PageableExtension].ToString());
                     if (pageableExtension != null && !method.Extensions.ContainsKey("nextLinkMethod") && !string.IsNullOrWhiteSpace(pageableExtension.NextLinkName))
```

Comment:
nextLinkMethod, indicates whether the method is the "next" method, nextMethodName is the name of the next method for a pageable operation, so they refer to difference things.

---
In reply to: [74296420](https://github.com/Azure/autorest/pull/1335#discussion_r74296420) [](ancestors = 74296420)